lib/fabrik/DockerInstance.js

```diff
@@ -48,17 +50,54 @@ class DockerInstance extends BaseInstance {
     return false;
   }
 
+  get platformContext() {
+    return this.inspectContainer()
+      .then(() => {
+        if (this.getEnvironment().context) {
+          return JSON.parse(this.getEnvironment().context);
+        } else {
+          throw new NotFound(`platform-context not found in docker environemt for instance ${this.guid}`);
```

Comment:
isnt it simple to just resolve the default value in else, rather than throwing `not found` and then resolving default value.
Also the err clause of NOT FOUND. Are you sure you want to do that, because thats the error from inspectContainer Call which might return back with container not found and for that you are returning default context value, which to me seems wrong. So you should not have any error handler but just send default value in else clause.

Reply:
In order to send default value we need to handle not found, as the 'context'  is not present for older containers.  So if we don't handle error in this case it will fail the operations where platform doesn't send context e.g. unbind and delete.

And in case where container not found itself then (even though context is set to default) in 'removeContainer' method itself it will thrwo error (https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/lib/fabrik/DockerInstance.js#L234).